src/main.rs

```diff
@@ -1,17 +1,19 @@
 #![warn(clippy::all)]
 
+mod api;
+
 use clap::{Parser, Subcommand};
 use cli_batteries::version;
-use tracing::info;
-use sqlx::{SqliteConnection, Connection};
+use sqlx::{Connection, SqliteConnection};
 use thiserror::Error;
+use tracing::info;
 
 #[derive(Parser)]
 struct Options {
     #[command(subcommand)]
     command: Commands,
 
-    #[arg(env="SITTER_CONNECTION_STRING")]
+    #[arg(env = "SITTER_CONNECTION_STRING")]
```

Comment:
Why not leave it at the default `CONNECTION_STRING` ?

Reply:
Ah, I didn't realize there was a default.

No strong reason.

Namespacing seems slightly easier&safer for end-users: all processes in a pipeline get the same environment and non-namespaced names might collide with the names other processes expect. No other tool is going to recognize CONNECTION_STRING, but for consistency we should either namespace or not namespace all the names we use.

I'm not particularly attached to this though, I'm happy to leave it CONNECTION_STRING and drop the SITTER_ prefix.